README.md

```diff
@@ -18,6 +19,7 @@ Each folder in this repository has two items within:
  - `README` - An explanation of everything about the data
  - `csv` - A folder with all relevant data in either CSV or TSV format
```

Comment:
Why do we need/want csv dirs before the parquet files? Maybe we can complicate this explanation a bit to allow for more variety? It's not like we're actually matching this comment.

Reply:
Maybe just clarify this?
Each folder in this repository except `parquet` has two items within:
 - `README` - An explanation of everything about the data
 - `csv` - A folder with all relevant data in either CSV or TSV format
 `parquet` instead contains a number of standalone `.parquet` files with different data formats